dashboard/app/models/pd/workshop.rb

```diff
@@ -315,13 +315,13 @@ def year
     sessions.order(:start).first.start.strftime('%Y')
   end
 
-  # returns the school year the summer workshop is preparing for, in
-  # the form "2019-2020", like application_year on Pd Applications
-  def summer_workshop_school_year
-    if local_summer?
-      y = year
-      "#{y}-#{y.to_i + 1}"
-    end
+  # Returns the school year the summer workshop is preparing for, in
+  # the form "2019-2020", like application_year on Pd Applications.
+  # The school year runs 6/1-5/31.
+  def school_year
+    return nil if sessions.empty?
+    y = year
+    sessions.order(:start).first.start.month >= 6 ? "#{y}-#{y.to_i + 1}" : "#{y.to_i - 1}-#{y}"
```

Comment:
Mixed feelings on shortening `year` to `y` just so that we can produce a very dense piece of code in the next line :)  

But this is still fairly readable so it's probably okay.

Reply:
I did that because calling `year` looks up the sessions every time, but now that you mention it, that lookup is by id, so this little improvement is probably not actually making much difference